web/toolbar.js

```diff
@@ -112,6 +112,17 @@ class Toolbar {
               : AnnotationEditorType.STAMP;
           },
         },
+        telemetry: () => {
+          eventBus.dispatch("reporttelemetry", {
+            source: this,
+            details: {
+              type: "editing",
+              data: {
+                action: "pdfjs.image.icon_click",
+              },
+            },
+          });
+        },
```

Comment:
If we ever want to add more telemetry reporting in this code, it seems that we'd have a bunch of duplicated boilerplate code with the current approach. So how about the following instead?
```suggestion
        telemetry: {
          type: "editing",
          data: { action: "pdfjs.image.icon_click" },
        },
```